site/pages/components/ordered-list.tsx

```diff
@@ -0,0 +1,373 @@
+import React from 'react';
+import {InlineMessage, toNewsKitIcon, UnorderedList} from 'newskit';
+import {Info as FilledInfo} from '@emotion-icons/material/Info';
+import {InlineCode} from '../../components/markdown-elements';
+import {UsageKind} from '../../components/usage-card';
+import {ComponentPageTemplate} from '../../templates/component-page-template';
+import {LayoutProps} from '../../components/layout';
+import {MetaStatus} from '../../components/meta/types';
+import {OrderedList} from '../../../src/ordered-list';
+import {getIllustrationComponent} from '../../components/illustrations/illustration-loader';
+import {IconFilledCircle} from '../../components/icons';
+import {Link} from '../../components/link';
+import {commonLogicalProps} from '../../components/component-api/common-logical-props';
+import {OverridesRowsProps} from '../../components/component-api';
+
+const IconFilledInfo = toNewsKitIcon(FilledInfo);
+
+const infoIcon = (
+  <IconFilledInfo
+    overrides={{
+      size: 'iconSize020',
+    }}
+  />
+);
+
+const commonPropsRows = [
+  {
+    name: 'children',
+    type: 'Array<React.ReactNode>',
+    description: (
+      <>
+        An array of react nodes or strings to populate the ordered list
+        component.
+        <br />
+        <br />
+        Note - if the node is a react element, a unique key must be given to
+        each element within the array.
+      </>
+    ),
+    required: true,
+  },
+];
+const commonOverridesRows = [
+  {
+    attribute: 'orderedList.spaceInline',
```

Comment:
We don't need `orderedList` prefix, you need to remove it from all overrides
```suggestion
    attribute: 'spaceInline',
```

Reply:
Done @mutebg